sdk/storage/azure-storage-blob/azure/storage/blob/_generated/aio/operations/_container_operations.py

```diff
@@ -1452,12 +1452,12 @@ async def list_blob_flat_segment(
         response_headers['x-ms-request-id']=self._deserialize('str', response.headers.get('x-ms-request-id'))
         response_headers['x-ms-version']=self._deserialize('str', response.headers.get('x-ms-version'))
         response_headers['Date']=self._deserialize('rfc-1123', response.headers.get('Date'))
-        deserialized = self._deserialize('ListBlobsFlatSegmentResponse', pipeline_response)
+        #deserialized = self._deserialize('ListBlobsFlatSegmentResponse', pipeline_response)
```

Comment:
This is one of the bigger challenges to figure out. Currently the deserialization process is out of our hands. We would need to add directives to the autorest code gen for the list_blobs_flat_segment API to _not_ deserialize the response payload. This should be possible by simply overwriting the output model to have no output. We then use the `cls` hook and do the deserialization ourselves.
In the case of the existing `list_blobs` API, this probably means manually using the existing msrest deserializer if we don't want to deal with the testing burden of validating the new deserializer for the old API.